src/zarr/api/asynchronous.py

```diff
@@ -817,19 +819,40 @@ async def create(
     shape : int or tuple of ints
         Array shape.
     chunks : int or tuple of ints, optional
-        Chunk shape. If True, will be guessed from `shape` and `dtype`. If
-        False, will be set to `shape`, i.e., single chunk for the whole array.
-        If an int, the chunk size in each dimension will be given by the value
-        of `chunks`. Default is True.
+        The shape of the array's chunks.
+        V2 only. V3 arrays should use `chunk_shape` instead.
+        Default values are guessed based on the shape and dtype.
     dtype : str or dtype, optional
         NumPy dtype.
+    chunk_shape : int or tuple of ints, optional
+        The shape of the Array's chunks (default is None).
+        V3 only. V2 arrays should use `chunks` instead.
+    chunk_key_encoding : ChunkKeyEncoding, optional
+        A specification of how the chunk keys are represented in storage.
+        V3 only. V2 arrays should use `dimension_separator` instead.
+        Default is ("default", "/").
+    codecs : Sequence of Codecs or dicts, optional
+        An iterable of Codec or dict serializations thereof. The elements of
+        this collection specify the transformation from array values to stored bytes.
+        V3 only. V2 arrays should use `filters` and `compressor` instead.
+        If no codecs are provided, default codecs will be used:
+        - For numeric arrays, the default is `BytesCodec` and `ZstdCodec`.
+        - For Unicode strings, the default is `VLenUTF8Codec`.
```

Comment:
Are the codecs here meant to be links to their API reference in numcodecs? If so, they are not working.

Reply:
I did not intend to create links. How would you link the API reference? Is there another way but to manually add markdown links like ```[`VLenUTF8Codec`](https://numcodecs.readthedocs.io/en/stable/other/vlen.html#vlenutf8)```
Do you think we should add them?

Reply:
For ease here, maybe lets leave the links for now and just put these in double backticks. We can add the links later in another PR